src/components/ButtonsGroup/ButtonContent.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { memo } from 'react';
+import { Text, Pressable, StyleSheet } from 'react-native';
+import { colors, radius, sizes } from '@magnetis/astro-tokens';
+
+import type { Callback } from './types';
+
+export interface ButtonContentProps {
```

Comment:
Pode extender de Pressable para não precisar redeclarar props que já são aceitas
```suggestion
export interface ButtonContentProps extends PressableProps {
```
Assim, testID, onPress podem ser omitidos e qualquer outra prop que seja necessário passar para Pressable, vai estar disponível.